contracts/test/eip155.ts

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { EIP155Tests__factory } from '../typechain-types/factories/contracts/tests';
+import { StaticJsonRpcProvider } from '@ethersproject/providers';
```

Comment:
This dependency already exists as `ethers.providers.StaticJsonRpcProvider`, so you can safely remove the extra installed package.